From 28f18534f131ed25d8fc2a568cf97e5122efe2a0 Mon Sep 17 00:00:00 2001
From: "John B. Wyatt IV" <jwyatt@redhat.com>
Date: Wed, 13 Aug 2025 11:41:57 -0400
Subject: [PATCH] Revert "tuna: Update man page with cpu_power command" for
 RHEL10

This reverts commit 84baeed49a3d3e0b6b20138cbfe5ecbcfdbc0d67.

The cpu_power command was removed to reduce customer confusion.
---
 docs/tuna.8 | 18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/docs/tuna.8 b/docs/tuna.8
index 6e30537..2423894 100644
--- a/docs/tuna.8
+++ b/docs/tuna.8
@@ -24,24 +24,6 @@ Log application details to file for given LOG-LEVEL
 Print DEBUG level logging details to console
 .SH "COMMANDS"
 .TP
-\fBtuna cpu_power\fR
-usage: tuna-cmd.py cpu_power [-h] (-c CPU-LIST)
-
-Set or query the idle (power) states of all or a selection of cpus in the CPU-LIST. Modifiers \fB\-c\fR may be used to pass the CPU-LIST of cpus to perform the command on.
-
-optional arguments:
-  -h, --help            show this help message and exit
-  -i, --idle-info       Print general idle information for the selected CPUs, including
-                        index values for IDLE-STATE.
-  -s, --status IDLE-STATE
-                        Print whether IDLE-STATE is enabled on the selected CPUs.
-  -d, --disable IDLE-STATE
-                        Disable IDLE-STATE on the selected CPUs.
-  -e, --enable IDLE-STATE
-                        Enable IDLE-STATE on the selected CPUs.
-  -c CPU-LIST, --cpus CPU-LIST
-                        CPU-LIST affected by commands
-.TP
 \fBtuna isolate\fR
 usage: tuna-cmd.py isolate [-h] (-c CPU-LIST | -S CPU-SOCKET-LIST | -N)
 
-- 
2.50.1

